docs/distribution/README.md

```diff
@@ -0,0 +1,566 @@
+# OCI Distribution
+
+To support [Notary v2 goals][notaryv2-goals], upload, persistence and discovery of signatures must be supported. Alternative designs were considered, as referenced in [persistance-discovery-options.md](./persistance-discovery-options.md).
+
+This document represents the current working prototype which:
+
+- Leverages [OCI Index][oci-index] to store Notary v2 Signatures
+- Implements `index.config` to align with the [OCI Artifacts][oci-artifacts] approach for artifact type differentiation within a registry.
+- Implements a referrer API to identify referenced artifacts. Such as what signatures refer to a specific container image.
+
+## Table of Contents
+
+- [Signature Persistence](#signature-persistence)
+- [Signature Discovery](#signature-discovery)
+- [Persisting Referrer Metadata (Signatures)](#persisting-referrer-metadata-signatures)
+- [Implementation](#implementation)
+- [Push, Discover, Pull Prototype](#push-discover-pull-prototype)
+
+## Signature Persistence
+
+Several [options for how to persist a signature were explored][signature-persistance-options] . We measure these options against the [goals of Notary v2][notaryv2-goals], specifically:
+
+- Maintain the original artifact digest and collection of associated tags, supporting existing dev through deployment workflows
+- Multiple signatures per artifact, enabling the originating vendor signature, public registry certification and user/environment signatures

```

Comment:
I don't think how to store the multiple signatures is discussed in this doc:
https://github.com/notaryproject/nv2/blob/prototype-1/docs/distribution/persistance-discovery-options.md

I assume for multiple signatures admin has to push multiple indexes?